app/Metadata/Extractor.php

```diff
@@ -82,7 +82,7 @@ public function extract(string $filename, string $type): array
 			$is_raw = true;
 		}
 
-		if (strpos($type, 'video') !== 0) {
```

Comment:
are you sure about that one ?

Reply:
Pretty sure... It's because of those crazy MPG files being identified by `mime_content_type` as `image/x-tga`. So I added `image/x-tga` to `$validVideoTypes`, fixed `file_type` to do what it was actually supposed to (so that it actually returns `'video'` for video files), and then replaced all these ad-hoc tests by the more robust checks based on the value returned by `file_type`.